scripts/ci/update_downstream.sh

```diff
@@ -46,25 +46,24 @@ mvn -B versions:set-property versions:commit \
   -Dproperty=dependency.acs-packaging.version \
   "-DnewVersion=${VERSION}"
 
-sed -i "s/.*RELEASE_VERSION=.*/    - RELEASE_VERSION=$RELEASE_VERSION/" .travis.yml
-sed -i "s/.*DEVELOPMENT_VERSION=.*/    - DEVELOPMENT_VERSION=$DEVELOPMENT_VERSION/" .travis.yml
+sed -i "s/.*RELEASE_VERSION=.*/    - RELEASE_VERSION=$RELEASE_VERSION/" .github/workflows/ci.yml
+sed -i "s/.*DEVELOPMENT_VERSION=.*/    - DEVELOPMENT_VERSION=$DEVELOPMENT_VERSION/" .github/workflows/ci.yml
 
 set +e
-echo "${TRAVIS_COMMIT_MESSAGE}" | grep '\[publish\]'
+echo "${COMMIT_MESSAGE}" | grep '\[publish\]'
 if [ "$?" -eq 0 ]
 then
   COMMIT_DIRECTIVES="[release][publish]"
 else
   COMMIT_DIRECTIVES="[release]"
 fi
 set -e
-
 # Commit changes
 git status
 git --no-pager diff pom.xml
 git add pom.xml
 git --no-pager diff pom.xml
-git add .travis.yml
+git add .github/workflows/ci.yml
```

Comment:
Are these 3 references to `.github/workflows/ci.yml` correct, given that the `RELEASE_VERSION` and `DEVELOPMENT_VERSION` are inside `.github/workflows/master_release.yml` instead?

Reply:
Yes because we need to update the `RELEASE_VERSION` and `DEVELOPMENT_VERSION` in `acs-community-packaging` (`.github/workflows/ci.yml`) and to retrieve the release and development versions from `acs-packaging` (`.github/workflows/master_release.yml`)